### PR TITLE
[venice] Introduce the Write operation and clean up the metrics

### PR DESCRIPTION
**Pull request by eolivelli**
- introduce a new WriteOp to write directly to Venice
- support AVRO keys for reads and for writes
- remove useless metrics

### CODE REVIEW

File: adapter-venice/src/main/resources/scripts/download.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+##########
+set -x -e
+HERE=$(realpath $(dirname $0))
+VENICETOOLSURL=https://github.com/datastax/venice/releases/download/ds-0.4.17-alpha-12/venice-admin-tool-all.jar
+BINDIR=$HERE/binaries
+rm -Rf $BINDIR
+mkdir $BINDIR
+pushd $BINDIR
+cd $BINDIR
+curl -L -O $AVROTOOLSURL
```
Comment (dave2wave) [inline adapter-venice/src/main/resources/scripts/download.sh:12]: Change to VENICETOOLSURL

File: adapter-venice/src/main/resources/scripts/create-store.sh [hunk 2]
```diff
@@ -0,0 +1,24 @@
+#bin/bash
+# This is an utility script to create the store in Venice.
+# Use ./download.sh in order to download the binaries needed to run this script.
+
+set -x -e
+HERE=$(realpath $0)
+cd $HERE
```
Comment (dave2wave) [inline adapter-venice/src/main/resources/scripts/create-store.sh:7]: I think you meant `echo` and not `cd`

File: adapter-venice/src/main/resources/scripts/create-store.sh [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+jar=binaries/*admin-tool-all*.jar
+storeName=$1
+url=http://localhost:5555
+clusterName=venice-cluster0
+keySchema=key.avsc
+valueSchema=value.avsc
+
+# create the store
+java -jar $jar --new-store --url $url --cluster $clusterName  --store $storeName --key-schema-file $keySchema --value-schema-file $valueSchema --hybrid-data-replication-policy NON_AGGREGATE
```
Comment (dave2wave) [inline adapter-venice/src/main/resources/scripts/create-store.sh:17]: It seems that `store1` was already present. Is there a better way to start venice with gradlew that will assure no `store1` is present?

**Review by dave2wave — CHANGES_REQUESTED** [review]
There are some small errors in a couple of scripts and a "start venice fresh" question.

File: adapter-venice/src/main/resources/scripts/create-store.sh (hunk 3 above)
Comment (eolivelli) [reply adapter-venice/src/main/resources/scripts/create-store.sh:17]: Unfortuantely for Venice Standalone there is still no "--wipe-data" command.

Comment (eolivelli) [reply adapter-venice/src/main/resources/scripts/create-store.sh:17]: you can wipe everything with this command on the venice root directory
`rm -Rf services/venice-standalone/zookeeperData services/venice-standalone/kafka-logs services/venice-standalone/rocksdb`

I am not adding this note here, I will open a feature request on Venice Standalone

**Comment by eolivelli** [comment]
@dave2wave I have addressed your comments on the scripts.

**Review by dave2wave — APPROVED** [review]
Tests out fine

**Pull request merged**